source/init/mus_initLBMPS_module.f90

```diff
@@ -59,13 +62,18 @@ subroutine mus_init_advRel_lbm_ps( relaxation, layout, compute )
 
     select case( trim(relaxation) )
     case( 'bgk' )
-      select case( trim(layout) )
-      case( 'flekkoy' )
+      select case( trim(order) )
+      case( 'first' )
```

Comment:
Assign function pointers acoording to `relaxation_options%variant` which is to be introduced in next merge.